com.avaloq.tools.ddk.xtext.format.test/build.properties

```diff
@@ -1,6 +1,7 @@
 source.. = src/,\
            src-gen/,\
-           resource/
+           resource/,\
+           xtend-gen/

```

Comment:
why the change?

Reply:
probably just a local workspace change. reverted.